templates/KnockoutSpa/ClientApp/components/app-root/app-root.ts

```diff
@@ -40,4 +40,4 @@ class AppRootViewModel {
     }
 }
 
-export default { viewModel: AppRootViewModel, template: require('./app-root.html') };
+export default { viewModel: AppRootViewModel, componentName: 'Home', template: require('./app-root.html') };
```

Comment:
This might of been accidental, Knockout is giving me a minor :
Any idea what that might be? @SteveSandersonMS 

```vendor.js?v=P985CJI4_0EhN8wTjSzDXZnMQt78MgU91Uz14XJKMk8:26089 
Uncaught Error: Unable to process binding 
"component: function (){return { name:route().page,params:route} }"
Message: No component name specified```